src/app/projects/new-project/new-project.component.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NewProjectComponent } from './new-project.component';
+import { MatBottomSheetRef } from "@angular/material";
+import { ProjectsService } from "../projects.service";
```

Comment:
There should be quotes instead of double quotes.

Reply:
done